### ISSUE TITLE
ImportError: cannot import name 'bdd_tester'

### ISSUE THREAD

**Issue by Bjwebb** [pwyf/data-quality-tester#34]
```
$ flask db upgrade
Usage: flask db upgrade [OPTIONS] [REVISION]

Error: While importing "DataQualityTester", an ImportError was raised:

Traceback (most recent call last):
  File "/home/bjwebb/opendataservices/pwyf/data-quality-tester/.ve/lib/python3.6/site-packages/flask/cli.py", line 235, in locate_app
    __import__(module_name)
  File "/home/bjwebb/opendataservices/pwyf/data-quality-tester/DataQualityTester/__init__.py", line 41, in <module>
    from DataQualityTester import commands, routes, models, views, lib
  File "/home/bjwebb/opendataservices/pwyf/data-quality-tester/DataQualityTester/commands.py", line 9, in <module>
    from DataQualityTester.models import SuppliedData
  File "/home/bjwebb/opendataservices/pwyf/data-quality-tester/DataQualityTester/models.py", line 13, in <module>
    from DataQualityTester.tasks import download_task
  File "/home/bjwebb/opendataservices/pwyf/data-quality-tester/DataQualityTester/tasks.py", line 7, in <module>
    from bdd_tester import bdd_tester
ImportError: cannot import name 'bdd_tester'
```

I'm assuming this is due to changes in the latest versions of the bdd_tester.

**Comment by andylolz** [comment]
Oops – yes, you’re right! Thanks for spotting it. I’ll take a look.

**Comment by andylolz** [comment]
Fixed in 25ac5c48578f375aa1d2a53207c046d588374c52.
